public/account_settings.php

```diff
@@ -63,33 +63,32 @@
 
 <?php
 if (isset($_POST['submit'])) {
+        empty($_POST["first_name"]) ?: $user->first_name = $_POST["first_name"];
+        empty($_POST["last_name"]) ?: $user->last_name = $_POST["last_name"];
+        empty($_POST["email"]) ?: $user->email = $_POST["email"];
 
-  empty($_POST["first_name"]) ?: $user->first_name = $_POST["first_name"];
-  empty($_POST["last_name"]) ?: $user->last_name = $_POST["last_name"];
-  empty($_POST["email"]) ?: $user->email = $_POST["email"];
+        if (!empty($_POST["student_id"])) {
+            if (ctype_digit($_POST["student_id"])) {
+                $user->student_id = $_POST["student_id"];
+            } else {
+                echo "The entered student ID - '".escape($_POST["student_id"]."' is not valid and should only contain digits.");
```

Comment:
why are my account_settings changes here?

Reply:
I ran the linter, I guess it fixed a bunch of stuff in your code